source/_header.html.erb

```diff
@@ -40,6 +40,7 @@
       <li><a href="/community">The Ember Community</a>
        <li><a href="/guidelines">Guidelines</a></li>
        <li><a href="http://github.com/emberjs/">Contribute (Github)</a></li>
+       <li><a href="https://discuss.emberjs.com/">Ember Discussion Forum</a></li>
```

Comment:
Thank you for the PR! 👍

I think if we add a Forum link we should probably also add the Slack community link, but then again, both are front and center on the community page, which is already in this dropdown. @jenweber @acorncom any thoughts?

Reply:
@MelSumner inclined to agree with you. While having both links easily findable in the menu would be handy, we need to curate the items in this list so we can help people clearly find major areas of the community. I'm inclined to say we don't want to merge this. @jenweber @locks ?

Reply:
I am kinda torn. On one hand, I don't want to clutter the nav. On the other hand, new users need those links the most and I don't know that they would find them as-is. I would like to do an informal UX test before making a decision. I'll show some new devs our site, and ask them to find the discussion board/slack chat.

Reply:
I just thought it was a good idea because the discussion forum seemed hidden away.

Reply:
@Dog2puppy do you think that if we simplified the Community page in general, it would be clearer for you? See #3156 & see what you think.